docs/oscar-to-carlos-bootstrap-conversion.md

```
@@ -0,0 +1,284 @@
+```
+
+---
+
+## The CARLOS Bootstrap Standard
+
+New and converted CARLOS pages use **Bootstrap 5** with an HTML5-compliant structure.
+The reference template is `docs/CARLOS_bootstrap_layout_Example.html`.
+
+### Key structural elements (in order)
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)